radicle-cli/src/commands/patch/create.rs

```diff
@@ -28,232 +23,126 @@ and description.
 -->
 "#;
 
-const REVISION_MSG: &str = r#"
-<!--
-Please enter a comment message for your patch update. Leaving this
-blank is also okay.
--->
-"#;
+pub fn handle_patch_comment(
+    message: &term::patch::Comment,
```

Comment:
Probably a good idea to rename this type and function to `Message` or `Summary`? It used to actually be a comment, but now it's title + description.